### PR TITLE
PullRequest for Issue1374: remove the .ui files for REIXS Acquaman

### PR DESCRIPTION
**Pull request by XuetaoLiu-CLS**
https://github.com/acquaman/acquaman/issues/1374

### CODE REVIEW

**Comment by dretrex** [comment]
@iainworkman can you review this?

File: source/ui/REIXS/REIXSXESSpectrometerControlEditor.h [hunk 1]
```diff
@@ -25,9 +25,14 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
...
-	class REIXSXESSpectrometerControlEditor;
-}
+class QLabel;
+class QComboBox;
+class QDoubleSpinBox;
+class QPushButton;
+
+//namespace Ui {
+//	class REIXSXESSpectrometerControlEditor;
+//}
```
Comment (iainworkman) [inline source/ui/REIXS/REIXSXESSpectrometerControlEditor.h]: Can this commented block be removed?

File: source/ui/REIXS/REIXSXESSpectrometerControlEditor.h [hunk 2]
```diff
@@ -68,8 +73,28 @@ private slots:
...
+	QComboBox* maskComboBox_;
+
+	QLabel *energyRangeLabel_;
+	QLabel *maskFeedbackLabel_;
+	QLabel *gratingFeedbackLabel_;
+	QLabel *energyFeedbackLabel_;
+
 private:
-	Ui::REIXSXESSpectrometerControlEditor *ui_;
+//	Ui::REIXSXESSpectrometerControlEditor *ui_;
```
Comment (iainworkman) [inline source/ui/REIXS/REIXSXESSpectrometerControlEditor.h]: Same as above re: commented code. Can this be removed?

**Comment by iainworkman** [comment]
Other than the comments this looks good.

**Pull request merged**